config/storage.yml

```diff
@@ -0,0 +1,36 @@
+test:
+  service: Disk
+  root: <%= Rails.root.join("tmp/storage") %>
+
+local:
+  service: Disk
+  root: <%= Rails.root.join("storage") %>
+  host: <%= ENV['LOCAL_STORAGE_HOST'] %>
```

Comment:
Does this mean we have to define this ENV or else it'll break? If so, that should be documented in setup instructions.

Reply:
Good catch, will document.